src/style.css

```diff
@@ -1 +1,25 @@
 /* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(images/star.svg);
+  background-position: center;
+  margin-right: 4px;
+  background-repeat: no-repeat;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1),
```

Comment:
The `.stars--1 .stars__star:nth-child(-n + 1)` and similar selectors are correct for applying styles based on a parent class; however, ensure that the specificity does not become too high, making it difficult to override if needed.